static/js/map/map.js

```diff
@@ -416,7 +416,7 @@ function createUserLocationMarker() {
     const lng = useStoredPosition ? pos.lng : serverSettings.centerLng
 
     var marker = L.marker([lat, lng]).addTo(markersNoCluster)
-    marker.bindPopup(`<div><b>${i8ln('My location')}</b></div>`, { autoPan: autoPanPopup() })
+    marker.bindPopup(function () { return `<div><b>${i8ln('My location')}</b></div>` }, { autoPan: autoPanPopup() })
```

Comment:
No need to use a function here, just HTML is allowed: https://leafletjs.com/reference-1.7.1.html#layer-bindpopup